addons/google_calendar/models/calendar.py

```diff
@@ -298,7 +298,7 @@ def _google_values(self):
             'start': start,
             'end': end,
             'summary': self.name,
-            'description': tools.html_sanitize(self.description) if not tools.is_html_empty(self.description) else '',
+            'description': tools.html_sanitize(self._get_customer_description()) if not tools.is_html_empty(self._get_customer_description()) else '',
```

Comment:
Could be more interesting that ``_get_customer_description`` returns a sanitized / validated / Markupized html so that callers don't have to deal with the output and can consider it is safe